recipes/sorted_nearest/meta.yaml

```
@@ -0,0 +1,49 @@
+{% set name = "sorted_nearest" %}
+{% set version = "0.0.17" %}
+{% set file_ext = "tar.gz" %}
+{% set hash_type = "sha256" %}
+{% set hash_value = "dc8c765ee7c56738fa5be16ec453e18978f1ecc7675832ab4c0cdf3a50115727" %}
+
+package:
+  name: '{{ name|lower }}'
+  version: '{{ version }}'
+
+source:
+  fn: '{{ name }}-{{ version }}.{{ file_ext }}'
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.{{ file_ext }}
+  '{{ hash_type }}': '{{ hash_value }}'
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record=record.txt
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+  host:
+    - python
+    - setuptools
+    - cython
+    - numpy
+  run:
+    - python
+    - cython
+    - numpy
```

http://conda-forge.org/docs/maintainer/knowledge_base.html?highlight=numpy#building-against-numpy
```suggestion
    - {{ pin_compatible('numpy') }}
```